src/main/java/com/github/maxopoly/WurstCivTools/ConfigParser.java

```diff
@@ -57,6 +54,10 @@ public void parseTools(ConfigurationSection config) {
 			WurstEffect effect;
 			switch (type) {
 			case "PYLONFINDER":
+				if (current.getBoolean("enabled",false) == false){
```

Comment:
Hmm yeah, but you can comment it out easily if needed. Also having this set to disabled by default might be very confusing when someone else uses it.